library/src/main/java/com/schibsted/spain/barista/internal/viewaction/GeneralSwipeWithPartiallyVisibleViewAction.kt

```diff
@@ -0,0 +1,134 @@
+package com.schibsted.spain.barista.internal.viewaction
+
+import android.support.test.espresso.PerformException
+import android.support.test.espresso.UiController
+import android.support.test.espresso.ViewAction
+import android.support.test.espresso.action.CoordinatesProvider
+import android.support.test.espresso.action.PrecisionDescriber
+import android.support.test.espresso.action.Press
+import android.support.test.espresso.action.Swipe
+import android.support.test.espresso.action.Swiper
+import android.support.test.espresso.action.ViewActions
+import android.support.test.espresso.matcher.ViewMatchers
+import android.support.test.espresso.util.HumanReadables
+import android.view.View
+import android.view.ViewConfiguration
+import org.hamcrest.Matcher
+
+object SwipeActions {
```

Comment:
Since this file contains two classes, one public and one private; how about naming the file after the public class instead of the private one? 

I.e. naming the file `SwipeActions`.